config/routes.rb

```diff
@@ -28,6 +28,10 @@
       end
 
       resources :versions, only: [:index, :show], format: :json
+      resources :versions, only: [:show] do
+        get 'raw', on: :member
+      end
```

Comment:
Why isn’t this part of the above `resources :versions` call?

It might also make sense to specify `format: false` here, since it doesn’t really make sense to ask for a particular content type on the `raw` endpoint:

```rb
get 'raw', on: member, format: false
```

That prevents a route like `/api/v0/versions/abc123/raw.json` from working — you’d have to remove the `.json` part.

Reply:
This was a leftover from copying the few usable bits over from my first pr on this issue. to start this one. Will cleanup.